rekon/README.md

```diff
@@ -0,0 +1,7 @@
+# Rekon
+
+Rekon is a [facade] for an abstract reconciliation engine.
```

Comment:
My French screams at the absence of cedilla (“façade”). I'll try to deal with it :face_with_peeking_eye: